modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -68,12 +68,12 @@ enum ImreadModes {
        IMREAD_ANYDEPTH             = 2,  //!< If set, return 16-bit/32-bit image when the input has the corresponding depth, otherwise convert it to 8-bit.
        IMREAD_ANYCOLOR             = 4,  //!< If set, the image is read in any possible color format.
        IMREAD_LOAD_GDAL            = 8,  //!< If set, use the gdal driver for loading the image.
-       IMREAD_GRAYSCALE_REDUCED_2  = 16, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/2.
-       IMREAD_COLOR_REDUCED_2      = 17, //!< If set, always convert image to the 3 channel BGR color image and the image size reduced 1/2.
-       IMREAD_GRAYSCALE_REDUCED_4  = 32, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/4.
-       IMREAD_COLOR_REDUCED_4      = 33, //!< If set, always convert image to the 3 channel BGR color image and the image size reduced 1/4.
-       IMREAD_GRAYSCALE_REDUCED_8  = 64, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/8.
-       IMREAD_COLOR_REDUCED_8      = 65  //!< If set, always convert image to the 3 channel BGR color image and the image size reduced 1/8.
+       IMREAD_REDUCED_GRAYSCALE_2  = 16, //!< If set, always convert image to the single channel grayscale image and the image size reduced 1/2.
```

Comment:
i wanted to rename newly added enum names but this causes error on building. how can i solve it ? @vpisarev , @mshabunin could you help me.